src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -345,6 +369,29 @@ export default {
 </script>
 
 <style lang="postcss" scoped>
+	.filter-pill {
```

Comment:
Some of the CSS here can be changed to tailwind as well, if not in the template (since we don't have all of the versions like hover in our compiled TW - we don't currently rely on JIT tailwind), here with `@apply`.